dev/source/docs/adding_simulated_devices.rst

```diff
@@ -0,0 +1,319 @@
+.. _adding_simulated_devices:
+
+===========================================
+Adding Simulated Peripherals to sim_vehicle
+===========================================
+
+Several devices can be simulated within SITL. These include Gimbals, Beacons, Rangefinders, VICON, RPM sensors,etc.
+
+Simulating On-Board OSD
+=======================
+
+When starting SITL, you can have it display a simulation of the integated OSD, if the autopilot includes one. Add the ``--osd`` option when starting SITL:
+
+::
+
+    sim_vehicle.py -v ArduPlane --console --map --osd
+
+.. note:: the OSD emulation displays OSD panel items and locations, but it does not allow multiple screens, nor units other than metric
+
+
+Adding a Gimbal
+===============
+
+SITL can simulate a virtual gimbal.
+
+.. note::
+
+   Gimbal simulation causes SITL to start sending
+   `MOUNT_STATUS <https://mavlink.io/en/messages/ardupilotmega.html#MOUNT_STATUS>`__
```

Comment:
This is no longer true; we send a `GIMBAL_DEVICE_ATTITUDE_STATUS` message.  And it only has a quaternion in it, so good luck! :-)

( @rmackay9 )

Reply:
can fix later once Randy updates